akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -65,12 +65,12 @@ class FileSinkSpec extends StreamSpec(UnboundedMailboxConfig) {
       }, create = false)
     }
 
-    "by default write into existing file" in assertAllStagesStopped {
+    "write into existing file without wiping existing data" in assertAllStagesStopped {
       targetFile { f ⇒
         def write(lines: List[String]) =
           Source(lines)
             .map(ByteString(_))
-            .runWith(FileIO.toPath(f))
+            .runWith(FileIO.toPath(f, Set(StandardOpenOption.WRITE, StandardOpenOption.CREATE)))
```

Comment:
No, this test case tests the old behaviour where old content is kept in the file, but new written at the start of the file. It needed to be adapted to have the old, unwanted behaviour.